README.md

```diff
@@ -1,32 +1,45 @@
 # Requiem
 
-![](https://raw.githubusercontent.com/Ladysnake/Requiem/master/requiem-logo-200x125.png)
+<p align="center">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset="requiem-logo-200x125_light.png" width="250">
+        <img src="requiem-logo-200x125.png" width="250">
+    </picture> 
+</p>
 
 [![Download](https://curse.nikky.moe/api/img/265729?logo)](https://curse.nikky.moe/api/url/265729) ![](https://jitpack.io/v/ladysnake/dissolution.svg) ![](https://img.shields.io/github/issues/ladysnake/requiem.svg)
 
+<p>
+
+[<img alt="Quilt Logo" width="400px" src="https://raw.githubusercontent.com/QuiltMC/art/master/contributed-art/doctor4t/quilt-exclusive-banner.png?raw=true" />](https://quiltmc.org/en/about/faq/#usage-questions)
+
+</p>
+
+
 Requiem is a Minecraft (Java Edition) mod aiming at removing the continuity break resulting from death,
 as well as adding new mechanics centered around soul dissociation and body possession.
 
-More information is available on the Ladysnake Website: https://ladysnake.github.io/wiki/requiem.
+More information is available on the Ladysnake Website: <https://ladysnake.github.io/wiki/requiem>.

```

Comment:
Does this do anything on GitHub ? In any case, might as well update the url to ladysnake.org

Reply:
To be real, I didn't even notice these Brackets. Seems like some auto-formationg magic by Milkdown, the Markdown Editor I used.

Reply:
Ah I see. It's a bit annoying to have those unnecessary changes though - have you tried using [HackMD](https://hackmd.io) instead ?

Reply:
I haven't. Thanks for suggesting it